docs/release-notes.md

```diff
@@ -9,14 +9,40 @@ description: >
 We recommend always using "tagged" versions of Ondat rather than "latest",
 and to perform upgrades only after reading the release notes.
 
-The latest tagged release is `2.9.0`. For
+The latest tagged release is `2.9.1`. For
 installation instructions see our
 [Install](/docs/install/) page.
 
-The latest CLI release is `2.9.0`, available from
+The latest CLI release is `2.9.1`, available from
 [GitHub](https://github.com/storageos/go-cli/releases) or containerised from
 [DockerHub](https://hub.docker.com/r/storageos/cli).
 
+## 2.9.1 - Release 2022-11-23
+
+This release improves stability and reduces resource consumption when creating many volumes at once.
+
+### New
+
+* Allow dataplane startup timeout to be configured
+* Increase control plane's default max log size to 25MiB and enable log compression
```

Comment:
25MiB? or 256?

Reply:
25 MiB, but there can be up to 10 log files, so the max total size is 250 MiB (though it'll likely be much smaller due to compression). 

I'll update this note to clarify